src/script_chainer/services/process_manager.py

```
@@ -173,6 +176,8 @@ class ProcessManager:
         target_process: 追踪的目标进程。
     """
 
+    _job_handle: int | None = None  # Windows Job Object 句柄（类级单例）
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)